### PR TITLE
Week2/doo/meeting

### PR DESCRIPTION
**Pull request by syeonnn**
> 12월 01일 대면 모임 진행을 위한 PR입니다.

- 푼 사람 : @syeonnn
- 푼 문제

  - [x] K진수에서 소수 개수 구하기]
  - [ ] [주차 요금 계산]

### CODE REVIEW

File: W2/doo/meeting/k진수에서 소수 개수 구하기.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+// for문으로 0이 나올때까지 돌려서 0이 아닌 수들만 배열에 저장
+// 배열에 담은 값들을 하나의 숫자로 변환
+// 소수 여부 판별
+
+// k_num = [];
+// while (n > 0) {
+//   k_num.push(n % k);
+//   n = Math.floor(n / k);
+// }
+// k_num.reverse();
```
Comment (Seo0H) [inline W2/doo/meeting/k진수에서 소수 개수 구하기.js:32]: 진수 변환에 대해 이해하고 그걸 코드로 작성 할 수 있다는게 정말 멋진것같아요..! 😮

File: W2/doo/meeting/주차 요금 계산.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    return Number(form_temp[0]) * 60 + Number(form_temp[1]);
+  }
+
+  records.forEach((record) => {
+    var [time, car, what] = record;
+
+    let inCar = {}; // 입차 목록
+    let outCar = {}; // 출차 목록
+
+    console.log(what === "IN");
```
Comment (Seo0H) [inline W2/doo/meeting/주차 요금 계산.js:15]: 이부분 제가 아까 스터디에선 var때문이 아닐까? 라고 했었는데 그 문제는 아니였네요!
`var [time, car, what] = record.split(" ")` 이렇게 해서 사용해야 아래 콘솔에서 true가 나오는 것 같아요.

그래도 var과 관련된 라이브 코테 문제에 대해 알려드리자면,, 아래 링크에서 해보실 수 있어요!
저도 구글링하면서 찾은 문제를 정리해둔거라, 참고가 되셨음 해요 ㅎㅎ

[codepan link](https://codesandbox.io/p/sandbox/interesting-ride-nvw7j7?file=%2Fsrc%2FApp.js%3A24%2C14&layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clpm9n1vi0006356khihwv6v4%2522%252C%2522sizes%2522%253A%255B70%252C30%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clpm9n1vh0002356kuk06ls6y%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clpm9n1vh0003356k2n5l9bl7%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clpm9n1vh0005356kqws9olz6%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B48.265978714812164%252C51.734021285187836%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clpm9n1vh0002356kuk06ls6y%2522%253A%257B%2522id%2522%253A%2522clpm9n1vh0002356kuk06ls6y%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clpm9n1vh0001356kzjgo2gpw%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252Fsrc%252Findex.js%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%255D%252C%2522activeTabId%2522%253A%2522clpm9n1vh0001356kzjgo2gpw%2522%257D%252C%2522clpm9n1vh0005356kqws9olz6%2522%253A%257B%2522id%2522%253A%2522clpm9n1vh0005356kqws9olz6%2522%252C%2522activeTabId%2522%253A%2522clpm9t6wc00ha356ku6hl0tv4%2522%252C%2522tabs%2522%253A%255B%257B%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%252C%2522id%2522%253A%2522clpm9t6wc00ha356ku6hl0tv4%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522path%2522%253A%2522%2522%257D%255D%257D%252C%2522clpm9n1vh0003356k2n5l9bl7%2522%253A%257B%2522tabs%2522%253A%255B%255D%252C%2522id%2522%253A%2522clpm9n1vh0003356k2n5l9bl7%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A10.052083333333343%257D)

**Review by Seo0H — COMMENTED** [review]
오늘 수고하셨습니다!~ 연두님 코드 보며 많이 배울 수 있었어요! 🔥 특히 진수 구하는걸 직접 구현하신게 정말 멋진 것 같아요.. 저도 공부를 더 열심히 해야겠네요.

File: W2/doo/meeting/주차 요금 계산.js (hunk 2 above)
Comment (syeonnn) [reply W2/doo/meeting/주차 요금 계산.js:15]: 우와!! 공유해주셔서 너무 감사합니다!!! 제가 확인이 너무 늦었네요ㅠㅠ
라이브 코테도 사실 처음 들어봤는데... 스터디를 통해서 도움을 정말 많이 받고 가는 것 같아요. 감사합니다~🙇‍♀️
이번 기회에 var에 대해 확실히 알고 가게 되네요!👍

**Pull request merged**